addons/udes_stock/security/ir.model.access.csv

```diff
@@ -2,3 +2,4 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_stock_location_category_manager,stock.location.category.manager,model_stock_location_category,base.group_system,1,1,1,1
 access_stock_location_category_user,stock.location.category.user,model_stock_location_category,base.group_user,1,0,0,0
 access_stock_mark_as_todo_wizard,stock_mark_as_todo_wizard,model_stock_picking_mark_todo,udes_security.group_trusted_user,1,1,1,1
+access_container_type,container.type,model_container_type,base.group_user,1,1,1,1
```

Comment:
Check with POs, but I think they might want these to be editable by Warehouse Manager and above.

Reply:
Checked:
> Hi. we will leave it as it is for now. We will do a permissions story later